pages/obitPage.js

```diff
@@ -0,0 +1,26 @@
+const { I } = inject();
+
+module.exports = {
+  // insert your locators and methods here
+  fields: {
+    messageBox: "textarea[name=_GuestbookForm_Message]",
+    nameField: "input[name=_GuestbookForm_From]",
+    emailField: "input[name=_GuestbookForm_Email]",
+  },
+  submitBtn:
+    "div.Box-sc-ucqo0b-0.cbNzut > div.Box-sc-ucqo0b-0.Flex-sc-d1l2vy-0.fsrkdX > button",
```

Comment:
Not sure if you are at this point yet in the onboarding project; try using the data-component for your selectors. 
data-component="GuestbookDecoySubmitButton". 

Other than that, LGTM!